src/common/SurgeStorage.h

```diff
@@ -88,7 +88,7 @@ extern float table_envrate_lpf alignas(16)[512],
              table_envrate_linear alignas(16)[512],
              table_glide_exp alignas(16)[512],
              table_glide_log alignas(16)[512];
-extern float table_note_omega alignas(16)[2][512];
+extern float table_note_omega alignas(16)[2][512];  // TODO: FIX SCENE ASSUMPTION?
```

Comment:
This [2] is not scenes. table_note_omega[0] maps to sin and table_note_omega[1] maps to cos. Can you remove the comment?

Reply:
Cool!